### DISCUSSION TITLE
How to use payload in different services (monorepo)

### DISCUSSION THREAD

**Discussion by eduarddotgg** [payloadcms/payload#8796] [General]
What would be the best approach to use payload in multiple services? 
For example one is an admin and rest api, another one is for more complex background jobs.
Also it's important that admin and REST API should execute hooks and for background jobs they should net be executed to avoid infinite loops (of course this can be done with env vars or context).

What can be better approach for migration files where should they be?

**Comment by nnWhisperer** [comment]
I really wonder about it, I'm thinking of taking a db-related background job part out of the main rest api app and I was wondering if payload cms installed on both services can still work fine, ie is that a supported usecase?

**Reply by nnWhisperer** [reply]
According to [another discussion](https://github.com/payloadcms/payload/discussions/6584), it is possible to make payload cms instances work fine while sharing the same db, hence, the usecase I'm reporting must be already supported.
